packager/mpd/base/representation_unittest.cc

```diff
@@ -234,7 +235,7 @@ TEST_F(RepresentationTest, CheckVideoInfoVp9CodecInWebm) {
 }
 
 TEST_F(RepresentationTest, CheckVideoInfoLegacyVp9CodecInWebm) {
-  FLAGS_use_legacy_vp9_codec_string = true;
+  absl::SetFlag(&FLAGS_use_legacy_vp9_codec_string, true);
```

Comment:
To avoid polluting global state in a test, please use `FlagSaver` from `packager/flag_saver.h`.  See examples of usage in `packager/file/udp_options_unittest.cc` and `packager/file/file_unittest.cc`.

Reply:
done